bigbluebutton-html5/imports/api/external-videos/server/methods/stopWatchingExternalVideo.js

```diff
@@ -0,0 +1,17 @@
+import { Meteor } from 'meteor/meteor';
+import { check } from 'meteor/check';
+import Logger from '/imports/startup/server/logger';
+import Meetings from '/imports/api/meetings';
+
+export default function startStream(credentials, options) {
+  const { meetingId, requesterUserId, requesterToken } = credentials;
+
+  Logger.info(' user sharing a new youtube video: ', credentials);
+
+  check(requesterUserId, String);
+  check(requesterToken, String);
+
+  Meetings.update({ meetingId }, { $set: { externalVideoUrl: null } });
```

Comment:
in `methods/` we typically push into Redis: https://github.com/bigbluebutton/bigbluebutton/blob/df55491a1a853d1f20061ece3ec9f4e659e12479/bigbluebutton-html5/imports/api/cursor/server/methods/publishCursorUpdate.js#L32